modules/apps/accessibility/accessibility-test/src/testFunctional/macros/AccessibilityMenu.macro

```diff
@@ -53,19 +53,19 @@ definition {
 			value1 = "\ENTER");
 	}
 
-	macro toggleUnderlinedLinksOff {
+	macro toggleAnOptionInTheAccessibilityMenuOff {
 		AccessibilityMenu.openAccessibilityMenu();
 
 		Uncheck.uncheckToggleSwitch(
-			key_toggleSwitchLabel = "Underlined Links",
+			key_toggleSwitchLabel = ${key_toggleSwitchLabel},
 			locator1 = "ToggleSwitch#ANY_TOGGLE_SWITCH");
 	}
 
-	macro toggleUnderlinedLinksOn {
+	macro toggleAnOptionInTheAccessibilityMenuOn {
```

Comment:
I think I'll change it to `toggleAnOptionOn ` so when we read the `.testcase` it looks more readable `AccessibilityMenu.toggleAnOptionOn`

What do you think?

Reply:
I agree with simplifying the macro name. I suggest: `AccessibilityMenu.enableSpecificOption` for readability.